src/ui/tray-icon.cpp

```diff
@@ -101,6 +102,11 @@ SeafileTrayIcon::SeafileTrayIcon(QObject *parent)
     connect(SeahubNotificationsMonitor::instance(), SIGNAL(notificationsChanged()),
             this, SLOT(onSeahubNotificationsChanged()));
 
+#ifdef Q_OS_WIN32
+    connect(this, SIGNAL(messageClicked()),
```

Comment:
mac is different, see more http://doc.qt.io/qt-4.8/qsystemtrayicon.html#messageClicked. Qt is always lazy with mac support. 
> This signal is emitted when the message displayed using showMessage() was clicked by the user.
> Currently this signal is not sent on Mac OS X.



Anyway you will need #543 and some extra code to handle with it.